common.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   az_count                      = local.environment == "prod" ? 2 : 1
   ingest_raw_cache_bucket_name  = "${local.environment}-dr2-ingest-raw-cache"
-  pre_ingest_step_function_name = "${local.environment_title}PreIngestStepFunction"
+  pre_ingest_step_function_name = "${local.environment_title}IngestStepFunction"
```

Comment:
Is it fine that there is no hyphen after "environment_title"?

Reply:
It's title case so it comes out as IntgIngestStepFunction which is ok I think.

Reply:
Why is this name title case but all the others are hyphen?

Reply:
Because I don't like consistency? Come to think of it, the role has a hyphen in it as well. Ok, I'll change it.